tests/test_app.py

```diff
@@ -106,6 +122,7 @@ def test_api_summary(db: Session, client: TestClient):
     data = resp.json()
     assert data["studies"] == 13
     assert data["non_parent_studies"] == 11  # excludes studies A and B
+    assert data["wfe_output_data_size_bytes"] == 55
```

Comment:
I recommend asserting something about the **original, related** field here for contrast (although we may eventually retire or repurpose that field... TBD).

```suggestion
    assert data["data_size"] == 110
    assert data["wfe_output_data_size_bytes"] == 55
```